acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -32,20 +30,10 @@
 
 install_packages_on(hosts, PACKAGES)
 
+# make sure install is sane, beaker has already added puppet and ruby
+# to PATH in ~/.ssh/environment
 agents.each do |agent|
-  case agent['platform']
-  when /windows/
-    arch = agent[:ruby_arch] || 'x86'
-    base_url = ENV['MSI_BASE_URL'] || "http://builds.puppetlabs.lan/puppet-agent/#{ENV['SHA']}/artifacts/windows"
```

Comment:
That's a problem we need to address in general. It looks like we could specify `dev_builds_url` as an option to `install_puppet_agent_dev_repo_on`.